apps/fluent-tester/src/TestComponents/Svg/SvgTest.tsx

```diff
@@ -58,41 +59,81 @@ const RectTest: React.FunctionComponent = () => {
   );
 };
 
+const RichTextInlineSvgTest: React.FunctionComponent = () => {
+  const [showInline, setShowInline] = React.useState(false);
+  const [isGreen, setIsGreen] = React.useState(true);
+
+  const toggleShowInline = () => {
```

Comment:
Should use React.useCallback

same as below

Reply:
Switched to useCallback